source/backups.rst

```diff
@@ -93,34 +96,75 @@ The schedule is specified in crontab format as explained in the
 `Operator
 Options <https://percona.github.io/percona-xtradb-cluster-operator/configure/operator>`__.
 
+.. _backups-manual:
+
 Making on-demand backup
 -----------------------
 
-To make on-demand backup, user should use YAML file with correct names
-for the backup and the PXC Cluster, and correct PVC settings. The
-example of such file is
-`deploy/backup/backup.yaml <https://github.com/percona/percona-xtradb-cluster-operator/blob/master/deploy/backup/backup.yaml>`__.
+To make an on-demand backup, the user should use a special backup configuration
+YAML file with the following contents:
 
-When the backup config file is ready, actual backup command is executed:
+* **backup name** in the ``name`` key,
+* **PXC Cluster name** in the ``pxcCluster``key,
+* **storage name** in the ``storageName`` key.
 
-::
+.. note:: The atual backup storage configuration is done in the ``backup``
+   section of the ``deploy/cr.yaml`` configuration file.
 
-   kubectl apply -f deploy/backup/backup.yaml
+The example of the backup configuration file is `deploy/backup/backup.yaml <https://github.com/percona/percona-xtradb-cluster-operator/blob/master/deploy/backup/backup.yaml>`_.
 
-**Note:** *Storing backup settings in a separate file can be replaced by
-passing its content to the ``kubectl apply`` command as follows:*
+When the backup config file is ready, actual backup command is executed:
 
-::
+.. code:: bash
 
-   cat <<EOF | kubectl apply -f-
-   apiVersion: pxc.percona.com/v1
-   kind: PerconaXtraDBClusterBackup
-   metadata:
-     name: backup1
-   spec:
-     pxcCluster: cluster1
-     storageName: fs-pvc
-   EOF
+   kubectl apply -f deploy/backup/backup.yaml
+
+.. note:: Storing backup settings in a separate file can be replaced by
+   passing its content to the ``kubectl apply`` command as follows:
+
+   .. code:: bash
+
+      cat <<EOF | kubectl apply -f-
+      apiVersion: pxc.percona.com/v1
+      kind: PerconaXtraDBClusterBackup
+      metadata:
+        name: backup1
+      spec:
+        pxcCluster: cluster1
+        storageName: fs-pvc
+      EOF
+
+.. _backups-private-volume:
+
+Storing backup on a private volume
```

Comment:
should we give an example for S3 config?
should we explain `deploy/cr.yaml` changes before `deploy/backup/backup.yaml` changes?